.github/workflows/release.yml

```diff
@@ -2,8 +2,8 @@ name: "Release"
 
 on:
   push:
-    branches:
-      - master
+#    branches:
```

Comment:
Do not modify pipelines.

Reply:
resolved, since it was needed temporary for testing